webclient/app/src/app/assets/styles/HeaderStyles.js

```diff
@@ -21,9 +21,10 @@ const AppHeaderStyles = makeStyles(theme => ({
         display: "flex",
         justifyContent: "left",
         paddingLeft: 20,
+        color: theme.palette.secondary.main,
```

Comment:
(Ich würde keine Sekundärfarbe hier verwenden sondern eine Primärfarbe)

Reply:
das kannst du gern alles anpassen, aber nicht in diesem PR

Reply:
Oh, ich sehe es gerade - ist ja garnicht backgroundColor
Ich nehme mal an, dass die normale Hintergrundfarbe der Gradient mit der Sekundärfarbe ist? Dann würde ich theme.palette.secondary.contrastText vorschlagen. Das ist das passende schwarz oder weiß für den Text. Außer du hast da was anderes vor

Reply:
geänder auf color: theme.palette.primary.contrastText. Aber eigentlich ist das nur im Template relevant...